source/models/row.py

```diff
@@ -0,0 +1,60 @@
+import os, json, uuid
+from commands_functions.schema_keys import Keys
+from outputs.exceptions import *
+from models.index import Index
+
+class Row:
+    def __init__(self, table, value = None, primary_key = None):
+        self.table = table
+        if value is not None:
+            self.__init_by_value(value)
+        elif primary_key is not None:
+            self.__init_by_primary_key()
+        else:
+            raise WrongInput(message = "the value and PK of the row is null")
+        
+    def __init_by_value(self, value):
+        self.value = value
+        self.primary_key = value[self.table.table_metadata.primary_key]
+
+    def __init_by_primary_key(self,primary_key):
```

Comment:
Why don’t you have a static method 
Row.load(primaru_key)
- read from file 
- New Row(data)

Reply:
if I understood ```__get_by_primary_key``` do so.

Reply:
ok done